pkg/activator/handler/concurrency_reporting_handler.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2018 The Knative Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package handler
+
+import (
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/knative/serving/pkg/autoscaler"
+	"github.com/knative/serving/pkg/reconciler"
+)
+
+func NewConcurrencyReportingHandler(statChan chan *autoscaler.StatMessage, reportChan <-chan time.Time, next http.Handler) *concurrencyReportingHandler {
+
+	channels := Channels{
+		// TODO(markusthoemmes): remove hardcoded value?
+		ReqChan:    make(chan ReqEvent, 100),
+		StatChan:   statChan,
+		ReportChan: reportChan,
+	}
+
+	// TODO(markusthoemmes): remove hardcoded value?
+	NewConcurrencyReporter("activator", channels)
```

Comment:
Are you going to look for this key in the autoscaler?  If so, you should make it a shared constant.  Alternatively you can add a new field to the autoscaler.Stat struct which is QueueLength.  Then you can count unique reporters that provide Concurrency and thereby drop the activator from the observed pod count without having to coordinate a name.